### ISSUE TITLE
Does anyone knows the preprocessing and postprocessing of YOLOv2 coco from onnx zoo?

### ISSUE THREAD

**Issue by Apisteftos** [onnx/onnx#5533]
I found a code about the tiny yolov2 onnx and followed the instructions from C# and darknet about the anchors and preprocessing. 

I am using the 

```
anchors = [0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828]

```
tooked from darknet. Now the postprocessing should be similar to tiny yolov2 which seems isn't. 

I am getting wrong class prediction + the bounding boxes are not in place.

**Comment by xadupre** [comment]
Is it possible to know more about the models you are comparing? (location, converter you used, ...)

**Comment by Apisteftos** [comment]
> Is it possible to know more about the models you are comparing? (location, converter you used, ...)

I am not converting them, I am using these from the onnx zoo. I am doing my thesis in  AI accelerators and testing the performance of the pre-trained onnx models from onnx zoo. I did that for almost all classification models and couple of them of the object detections & image segmentation, but they have huge problems. 

Problems:
-  Incomplete information ( preprocessing, postprocessing alogrithms, mAP from object detection model does not corresponds to the real numbers. Sometime is higher or less than what is documented )
- GPU acceleration (some convolution nodes exceeding the theoretical peak  TFOPs during the inference, which is good, but I don't know how to estimate it and maybe no one do it. )
- Profiling (missing data to estimate the performance , i.e. qlinearconv, MaxPool the are not extracting the weights in the profiled data, which I need them.)

Tested from Object Detection:

- Yolo v4
- Yolo v3
- Tiny Yolo V3
- Tiny Yolo V2
- SSD
- Faster R-CNN and Mask R-CNN ( which both I should take them out because they had very high performance in 2 nodes. For example 480 node and node 490 convolution operator have for batch size 1, 600TFLOs and 1.3 PFLOPs )
- Yolo V2 attempted but with no results (missing information of preprocessing and mostly for postprocessing algorithms)
